### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by brafdlog**

### CODE REVIEW

**Review by baruchiro — APPROVED** [review]
OK, we still need to work on it, for now, it only for building from source.

Depend on the milestones, consider adding this link: https://github.com/brafdlog/budget-tracking/actions?query=branch%3Amaster+workflow%3ABuild%2FRelease+is%3Asuccess+event%3Apush

To get the latest production artifact.

**Comment by brafdlog** [comment]
Its a work in progress, didnt finish yet.
When i wanted to update the steps for setting up ynab i saw that it would be really hard to do for an end user so i am working on that and will update those steps in the readme.

**Comment by baruchiro** [comment]
Now I see it is WIP

File: README.md [hunk 1]
```diff
@@ -10,83 +13,76 @@ In addition, it can automatically set an expense category for transactions accor
...
-- Initial implementation of creating transactions in a google spreadsheet of your choice
-- A very basic UI for configuration
-- Most of the configuration is currently in an encrypted JSON file and options that are not available in the ui can be changed by updating the JSON manually.
 
-## Initial setup
+## Running the app
+You can download the app from [here](https://github.com/brafdlog/budget-tracking/actions?query=branch%3Amaster+workflow%3ABuild%2FRelease+is%3Asuccess+event%3Apush)
+
+### Initial setup
+Before running the app for the first time you will need to set up the accounts you want to fetch data from (importers).
```
Comment (baruchiro) [inline README.md:27]:
```suggestion
The first time you run the app, you will need to set up the accounts you want to fetch data from (importers).
```

**Review by baruchiro — APPROVED** [review]

**Pull request merged**